frontend/src/components/WalletRow.vue

```diff
@@ -42,32 +42,24 @@
         </div>
       </div>
     </div>
+    <div class="q-py-sm q-mt-md row justify-center items-center bg-grey-2 cursor">
+      <router-link :class="{ 'no-text-decoration': true, 'dark-toggle': true }" :to="{ name: 'sent' }">{{
+        $t('WalletRow.send-history')
+      }}</router-link>
+    </div>
```

Comment:
A few comments here:
1. The whole thing looks like the button, but I can only click the text. Let's make the whole thing clickable
2. Let's add some change on hover, maybe the gold outline like we do when hovering the "Send" and "Receive" boxes